packages/components/src/form/docs/example.js

```diff
@@ -77,8 +77,11 @@ export default () => (
 				</Button>
 				<br />
 				<br />
-				Values: { JSON.stringify( values ) }<br />
-				Errors: { JSON.stringify( errors ) }<br />
+				<h3>Return data:</h3>
+				<pre>
+					Values: { JSON.stringify( values ) }<br />
+					Errors: { JSON.stringify( errors ) }
+				</pre>
```

Comment:
Was this change/commit intended to be included?

Reply:
Not at all. I never touched this file. Probably I did something wrong with the pull request or the branch... I will remove it.

Reply:
It's probably best to remove the commit entirely - you can do so using `git rebase` (I like using it in interactive mode `-i`)